cmake/kokkos_functions.cmake

```diff
@@ -230,9 +228,7 @@ MACRO(kokkos_import_tpl NAME)
   # I have still been getting errors about ROOT variables being ignored
   # I'm not sure if this is a scope issue - but make sure
   # the policy is set before we do any find_package calls
-  IF(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12.0")
-    CMAKE_POLICY(SET CMP0074 NEW)
-  ENDIF()
+  CMAKE_POLICY(SET CMP0074 NEW)
```

Comment:
I actually wonder if we still need that guy

Reply:
https://cmake.org/cmake/help/latest/policy/CMP0074.html still says
> [...] 
This policy was introduced in CMake version 3.12. CMake version 3.19.2 warns when the policy is not set and uses OLD behavior. Use the cmake_policy() command to set it to OLD or NEW explicitly.

Reply:
Look at the comment above that block of code.  The policy is being set in the main CMakeLists.txt of the package